packages/contracts-bedrock/tasks/update-registry.ts

```diff
@@ -0,0 +1,71 @@
+import fs from 'fs'
+import path from 'path'
+
+import { task } from 'hardhat/config'
+
+/**
+ * Updates the ChainRegistry contract by claiming new deployments
+ * and registering new contract addresses for existing deployments
+ */
```

Comment:
What do you think about migrating this deploy script to foundry?

Reply:
Should be trivial. What directory should it go in?